lib/react.dart

```diff
@@ -1186,6 +1194,23 @@ mixin TypedSnapshot<TSnapshot> {
 
 /// Creates a ReactJS virtual DOM instance (`ReactElement` on the client).
 abstract class ReactComponentFactoryProxy implements Function {
+  /// The JS component factory used by this factory to build [ReactElement]s.
+  ///
+  /// Deprecated: Use [React.createElement()] instead and pass in [type] as
+  /// the first argument, followed by `props` and `children`.
+  ///
+  /// Before:
+  /// ```
+  ///   YourReactComponentFactoryProxy.reactComponentFactory(props, children);
+  /// ```
+  ///
+  /// After:
+  /// ```
+  ///   React.createElement(YourReactComponentFactoryProxy.type, props, children);
+  /// ```
+  @Deprecated('6.0.0')
+  ReactJsComponentFactory reactComponentFactory;
```

Comment:
Can this be removed with the latest changes? This doesn't seem to be used anywhere, and shouldn't be added to the public API if not necessary

Reply:
it would be a breaking change to remove that public field, hence the deprecation... or would it not be breaking for some reason?

Reply:
@greglittlefield-wf ^

Reply:
I agree with Keal - this would be a breaking change.

Reply:
wait a second, this was Added, gregs probably correct, i added this here before i did a few other things, i think this should be removed. and it wouldnt be a break as it exists on all the other proxys themselves.

Reply:
this was already removed, this is outdated...